tests/kafkatest/tests/benchmark_test.py

```diff
@@ -44,25 +52,35 @@ def __init__(self, test_context):
         self.target_data_size = 128*1024*1024
         self.target_data_size_gb = self.target_data_size/float(1024*1024*1024)
 
-    @parametrize(acks=1, topic=TOPIC_REP_ONE, num_producers=1, message_size=DEFAULT_RECORD_SIZE)
-    @parametrize(acks=1, topic=TOPIC_REP_THREE, num_producers=1, message_size=DEFAULT_RECORD_SIZE)
-    @parametrize(acks=-1, topic=TOPIC_REP_THREE, num_producers=1, message_size=DEFAULT_RECORD_SIZE)
-    @parametrize(acks=1, topic=TOPIC_REP_THREE, num_producers=3, message_size=DEFAULT_RECORD_SIZE)
-    @matrix(acks=[1], topic=[TOPIC_REP_THREE], num_producers=[1], message_size=[10, 100, 1000, 10000, 100000])
-    def test_producer_throughput(self, acks, topic, num_producers, message_size):
+    def setUp(self):
+        self.zk.start()
+
+    def start_kafka(self, interbroker_security_protocol):
+        self.kafka = KafkaService(
+            self.test_context, self.num_brokers, 
+            self.zk, interbroker_security_protocol=interbroker_security_protocol, topics=self.topics)
+        self.kafka.start()
+
+    @parametrize(acks=1, topic=TOPIC_REP_ONE, num_producers=1, message_size=DEFAULT_RECORD_SIZE, security_protocol='PLAINTEXT')
```

Comment:
I think we can use default values on the method to avoid having so many parameters to all these `@parametrize` calls, e.g. make `test_producer_throughput` use `num_producers=1, message_size=DEFAULT_RECORD_SIZE, security_protocol='PLAINTEXT'` so the `@parametrize` calls only need to specify values they override.